### ISSUE TITLE
Destroy or manually mount units

### ISSUE THREAD

**Issue by Ricdez** [Anuken/Mindustry-Suggestions#4143]
### Describe the content or mechanics you are proposing.

I'm stuck in erekir split bcuz I can't destroy my units or launch them to the other side of the wall. My units are already full capacity thus I can't create more. Add a function where you can manually mount the units to fabricators, reconstructors and drivers.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Better controls

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by yesC48** [comment]
in mobile you can force your units into payload conveyors via double tapping. in PC I don't know because I don't own one

**Comment by Devilsknock** [comment]
Yeah I'm not sure if you can manually mount units on PC but being able to destroy them would be very nice. A delete command like Age of Empires has would be nice. Considering it's very easy to have your factories get stuck due to unit caps, it's kind of weird there currently isn't a way to fix that besides deleting and remaking them.

**Comment by Bottinator22** [comment]
PC, ctrl click the payload block while on top of it
If collisions, do it on a payload conveyor
